transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -71,6 +93,105 @@ protected AbstractEpollUnsafe newUnsafe() {
     }
 
     /**
+     * @see {@link #spliceTo(AbstractEpollStreamChannel, int, ChannelPromise)}
+     */
+    public final ChannelFuture spliceTo(final AbstractEpollStreamChannel ch, final int len) {
+        return spliceTo(ch, len, newPromise());
+    }
+
+    /**
+     * Splice from this {@link AbstractEpollStreamChannel} to another {@link AbstractEpollStreamChannel}.
+     *
+     * Please note:
+     * <ul>
+     *   <li>both channels need to be registered to the same {@link EventLoop}, otherwise an
+     *   {@link IllegalArgumentException} is thrown. </li>
+     *   <li>{@link EpollChannelConfig#getEpollMode()} must be {@link EpollMode#LEVEL_TRIGGERED} for this and the
+     *   target {@link AbstractEpollStreamChannel}</li>
+     * </ul>
+     *
+     */
+    public final ChannelFuture spliceTo(final AbstractEpollStreamChannel ch, final int len,
+                                        final ChannelPromise promise) {
+        if (ch.eventLoop() != eventLoop()) {
+            throw new IllegalArgumentException("EventLoops are not the same");
```

Comment:
When it's a sentence, it ends with a dot. Otherwise it doesn't.  In this case, it has to end with a dot.  Our exception message usually is not a sentence so they did not end with a dot.